hub/saml-engine/src/test/java/uk/gov/ida/hub/samlengine/validation/country/EidasAttributeStatementAssertionValidatorTest.java

```diff
@@ -88,6 +90,26 @@ public void shouldThrowIfAttributeValueHasInvalidSchemaType() {
         validator.validate(assertion);
     }
 
+    @Test(expected = SamlTransformationErrorException.class)
+    public void shouldThrowIfAttributeValueHasNullSchemaType() {
+        XMLObject xmlObject = mock(XMLObject.class);
+        when(firstName.getAttributeValues()).thenReturn(List.of(xmlObject));
+        when(xmlObject.getSchemaType()).thenReturn(null);
+
+        validator.validate(assertion);
+    }
+
+    @Test(expected = SamlTransformationErrorException.class)
+    public void shouldNotThrowIfAttributeValueHasNullSchemaNamespace() {
+        XMLObject xmlObject = mock(XMLObject.class);
+        when(firstName.getAttributeValues()).thenReturn(List.of(xmlObject));
+        when(xmlObject.getSchemaType()).thenReturn(new QName(null, CurrentGivenName.TYPE_LOCAL_NAME, IdaConstants.EIDAS_NATURUAL_PREFIX));
+        when(firstName.getNameFormat()).thenReturn("urn:oasis:names:tc:SAML:2.0:attrname-format:uri");
```

Comment:
There's a SAML constant called `URI_REFERENCE` for `"urn:oasis:names:tc:SAML:2.0:attrname-format:uri"` in `org.opensaml.saml.saml2.core.Attribute` class